go/lint-project.sh

```diff
@@ -40,7 +40,7 @@ if [[ "$OS_NAME" != "windows" ]]; then
 fi
 
 # Reject moovfinancial dependencies in moov-io projects
-org=$(basename $(dirname $(pwd)))
+org=$(head -n 1 "./go.mod" | awk -F'/' '{print $2}')
```

Comment:
Since I have all my projects in `~/moov`, if I run `make check` locally, `org` will be set to `moov`.

This change would set the org name based on the module name in `go.mod`.
With `module github.com/moovfinancial/webhooks`, `org = moovfinancial`

Reply:
Yea. I wonder if `go mod why | head -n1` is safer. That has the Go command hook into `go.mod` and parse it. A bit safer as think the `module ...` line can be elsewhere in the file.

Reply:
ohhh `go mod why` was exactly what i was looking for originally; I'll change this to:
```bash
> go mod why | awk -F'/' '{print $2}'
moovfinancial
```